app/views/group_mailer/invite_user.text.erb

```diff
@@ -1,9 +1,5 @@
 Metadata Management Tool Invitation
 
-<%= @invite.user_first_name %>, <%= @invite.manager_name %> from <%= @invite.provider %> would like to add you to a group using NASA's Metadata Management Tool (MMT). To do so, MMT needs to access your Earthdata Login profile. Please authorize MMT to access your profile by clicking the link below. Thank you!
-
-Authorize MMT: <%= accept_invite_url(token: @invite.token) %>
-
-If you do not have an Earthdata Login profile, please create one prior to authorizing MMT.
+<%= @invite.user_first_name %>, <%= @invite.manager_name %> from <%= @invite.provider %> would like to add you to a group using NASA's Metadata Management Tool (MMT). To do so, MMT needs you to create an Earthdata Login profile. To do so click on the link below. Thank You!
```

Comment:
have you spoken about this language with Alicia? I am wondering if we should thinking about adding language (possibly at a later date) with information about Launchpad requirements as well.

Reply:
Yeah I talked with Alicia yesterday and she said to just disable this stuff for now because things will with Launchpad